src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private static final int MAX_NUMBER_OF_BALLS = 100;
+
+    @Override
```

Comment:
The getRandomColor() method should be removed from Lottery class and used from the ColorSupplier instance mentioned earlier.

Reply:
According to task: "create a Lottery class with getRandomBall() method". Such actions will be contrary to this task condition.